packages/@sanity/cli/src/actions/init-project/initProject.ts

```diff
@@ -202,9 +243,189 @@ export default async function initSanity(
+          .replace(':basePath:', studioPath)
+      )
+    }
+
+    const sanityCliPath = path.join(workDir, `sanity.cli.${fileExtension}`)
+    await writeOrOverwrite(sanityCliPath, sanityCliTemplate)
+
+    // write sanity folder files
+    const writeSourceFiles = async (
+      files: Record<string, string | Record<string, string>>,
+      folderPath?: string
+    ) => {
+      for (const [filePath, content] of Object.entries(files)) {
+        // check if file ends with full stop to indicate it's file and not directory (this only works with our template tree structure)
+        if (filePath.includes('.') && typeof content === 'string') {
+          await writeOrOverwrite(
+            path.join(workDir, 'sanity', folderPath || '', `${filePath}${fileExtension}`),
+            content
+          )
+        } else {
+          await fs.mkdir(path.join(workDir, 'sanity', filePath), {recursive: true})
+          if (typeof content === 'object') {
+            await writeSourceFiles(content, filePath)
+          }
+        }
+      }
+    }
+
+    // ask what kind of schema setup the user wants
+    const templateToUse = unattended ? 'clean' : await promptForNextTemplate(prompt)
+
+    await writeSourceFiles(sanityFolder(useTypeScript, templateToUse))
+
+    // set tsconfig.json target to ES2017
+    const tsConfigPath = path.join(workDir, 'tsconfig.json')
+
+    if (useTypeScript && existsSync(tsConfigPath)) {
+      const tsConfigFile = readFileSync(tsConfigPath, 'utf8')
+      const config = evaluate(tsConfigFile)
+
+      if (config.compilerOptions.target?.toLowerCase() !== 'es2017') {
+        config.compilerOptions.target = 'ES2017'
+
+        const newConfig = patch(tsConfigFile, config)
+        await fs.writeFile(tsConfigPath, Buffer.from(newConfig))
+      }
+    }
+
+    const appendEnv = unattended ? true : await promptForAppendEnv(prompt, envFilename)
+
+    if (appendEnv) {
+      await createOrAppendEnvVars(envFilename, detectedFramework, {
+        log: true,
+      })
+    }
+
+    const {chosen} = await getPackageManagerChoice(workDir, {interactive: false})
+
+    await installNewPackages(
+      {
+        packageManager: chosen,
+        packages: ['@sanity/vision@3', 'sanity@3', '@sanity/image-url@1'],
+      },
+      {
+        output: context.output,
+        workDir,
+      }
+    )
+
+    // will refactor this later
+    const execOptions: CommonOptions<'utf8'> = {
+      encoding: 'utf8',
+      env: getPartialEnvWithNpmPath(workDir),
+      cwd: workDir,
+      stdio: 'inherit',
+    }
+
+    if (chosen === 'npm') {
+      await execa('npm', ['install', 'next-sanity@4'], execOptions)
+    } else if (chosen === 'yarn') {
+      await execa('npx', ['install-peerdeps', '--yarn', 'next-sanity@4'], execOptions)
+    } else if (chosen === 'pnpm') {
+      await execa('npx', ['install-peerdeps', '--pnpm', 'next-sanity@4'], execOptions)
```

Comment:
This is breaking `pnpm` on windows. In `next-sanity` we've used to recommend using `install-peerdeps` for a while: https://github.com/sanity-io/next-sanity#pnpm 

But as pnpm, [starting in v8, auto installs peers](https://pnpm.io/npmrc#auto-install-peers) we're soon going to remove this recommendation.

Thus I recommend we use the regular install method here, as it fixes the windows issue:
```suggestion
      await execa('pnpm', ['install', 'next-sanity@4'], execOptions)
```